act-rules-format/act-rules-format.bs

```diff
@@ -433,6 +440,20 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </aside>
 
+<aside class=example>
+  <header>Example applicability of a rule with visual headings as a test target</header>
+  <blockquote>
+    <p>The rule applies to any HTML element that is styled as a heading.</p>
+    <p>Since this applicability is subjective, it requires a glossary definition for "styled as a heading".</p>
+    <p>Styled as a heading defintion - An element is considered to be styled as a heading when it exhibits at least 2 of the following characteristics.</p>
+    <ul>
+      <li>Element text is visually distinct from nearby text, such as through the use of a larger font-size or bolding.</li>
+      <li>Element is clearly delineated from surrounding text, either through the use of spacing or other visual effects.</li>
+      <li>The element text broadly describes the content that follows it in the reading order of the page.</li>
+    </ul>
+  </blockquote>
+</aside>
```

Comment:
I tried to create an example of what form these types of definitions might take. In the format above, I left i up to the definition author to determine the logic behind the classification (i.e., it doesn't always have to be meet 3 of X criteria to apply). For example, I considered making the logic here that a heading should be at least one of the first 2 bullet points and then must meet the 3rd.

Reply:
I haven't made any changes to the format yet, but here are some of my initial responses to the questions we raised at the last meeting:

> What if we have a test target that doesn't meet the definition, but that we would still like to have as a test target? The example given was a heading that only fulfills one or none of the conditions above?

I think in all cases this would mean that the definition requires further expansion/editing. We already have processes in place to handle this, and I think as long as we stick to them we will be fine. I am a bit concerned that we will open the door to "cascading" complexity if we try to make our definitions cover too much ground, but I don't have any methods for preventing that at the moment. 

> What if we want to specify what something isn't? 

With our current conversation, we have discussed (and I believe at least initially agreed) to allow more complex logic inside of the definitions. In my opinion, this means usages of the standard `and`, `or`, and `not` operators will all be allowed when writing these definitions.